lasagne/tests/layers/test_conv.py

```diff
@@ -191,7 +197,7 @@ def test_defaults(self, Conv2DImpl, DummyInputLayer,
                 W=kernel,
                 **kwargs
             )
-            actual = layer.get_output_for(input).eval()
+            actual = get_output(layer, input).eval()
```

Comment:
Hmm, why this? The original version makes clear we're just propagating through this single layer to test its implementation of `get_output_for()`.